powerai_serverside/train_nasnet.py

```diff
@@ -0,0 +1,29 @@
+from keras import backend as K
+from keras.utils import multi_gpu_model
+from keras.models import Model
+from keras.layers import Dense
+from keras.applications.nasnet import NASNetMobile
+from keras.callbacks import ReduceLROnPlateau, TensorBoard, ModelCheckpoint
+from datasequence import DataSequence
+import pandas as pd
+import sys
+
+def euc_dist_keras(y_true, y_pred):
+    return K.sqrt(K.sum(K.square(y_true - y_pred), axis=-1, keepdims=True))
+
+model = NASNetMobile()
```

Comment:
comments here would go a long way in helping maintainers understand whats going on